config.ts

```diff
@@ -68,7 +68,8 @@ const ENV = {
       (domain) => [`https://${domain}`, `http://${domain}`, domain]
     ),
     enableTransactionFrames: true,
-    betaGroupChatUrl: "https://converse.xyz/group-invite/NfrIwpnhZdRCx2ZyGCg-S",
+    alphaGroupChatUrl:
+      "https://converse.xyz/group-invite/NfrIwpnhZdRCx2ZyGCg-S",
```

Comment:
this url shows up as not found, not sure if it was supposed to be updated as well